### PR TITLE
chore(): upgrade version fastify 4.x

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Upgrade version fastify 4.x

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

To be added to NestJS v9 😻 (To be moved to the `9.0.0` branch)

### CODE REVIEW

**Comment by tugascript** [comment]
Shouldn't you have to update the plugins as well? Those versions are deprecated.

fastify-cors -> @fastify/cors 
fastify-multipart -> @fastify/multipart

**Comment by Tony133** [comment]
I haven't updated them because they have already been integrated into the v9 branch in this PR (see here https://github.com/nestjs/nest/pull/9507) 😁. It was just missing to update the version of fastify because it came out a couple of days ago.

**Comment by kamilmysliwiec** [comment]
Can you use the `9.0.0` branch as a target branch?

File: package.json [hunk 1]
```diff
@@ -94,12 +94,8 @@
...
     "@types/mocha": "9.1.1",
-<<<<<<< HEAD
-    "@types/node": "17.0.36",
-=======
     "@types/node": "17.0.38",
     "@types/redis": "4.0.11",
->>>>>>> 663eeb85b (chore(deps-dev): bump @types/node from 17.0.36 to 17.0.38)
     "@types/reflect-metadata": "0.1.0",
     "@types/sinon": "10.0.11",
     "@types/supertest": "2.0.12",
```
Comment (jmcdo29) [inline package.json:101]: Remove the git conflict markers and resolve the conflict please

File: package.json [hunk 2]
```diff
@@ -130,15 +126,11 @@
...
     "fancy-log": "2.0.0",
-<<<<<<< HEAD
-    "fastify": "3.29.0",
-=======
     "fastify": "4.0.0",
     "fastify-cors": "6.1.0",
     "fastify-formbody": "5.3.0",
     "fastify-multipart": "5.4.0",
     "fastify-static": "4.7.0",
->>>>>>> 1d55a14e9 (chore(): upgrade version fastify 4.x)
```
Comment (jmcdo29) [inline package.json]: Remove the git conflict markers and resolve the conflict please

**Review by jmcdo29 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**